pkg/generator/generator_test.go

```diff
@@ -71,6 +71,7 @@ func TestGenMain(t *testing.T) {
 const handlerExp = `package stub
 
 import (
+	"context"
```

Comment:
a new line between 74 and 75.

We usually group internal go pkg imports as the first group, the current repo pkg imports as the second, and other external imports as the third group.

Reply:
Done